server/build/plugins/nextjs-ssr-import.js

```diff
@@ -10,13 +10,15 @@ export default class NextJsSsrImportPlugin {
 
   apply (compiler) {
     compiler.plugin('compilation', (compilation) => {
-      compilation.mainTemplate.plugin('require-ensure', (code) => {
+      compilation.mainTemplate.plugin('require-ensure', (code, chunk) => {
         // Update to load chunks from our custom chunks directory
-        const chunksDirPath = join(this.dir, this.dist, 'dist')
+        const baseDirPath = join(this.dir, this.dist, 'dist')
+        const pagePath = join(this.dir, this.dist, 'dist', dirname(chunk.name))
```

Comment:
Here we can use `/` instead of `this.dir, this.dist, 'dist'`. 
But the code we've is easy to read and gives us almost no overhead.